### DISCUSSION TITLE
Question on Experimental Design

### DISCUSSION THREAD

**Discussion by josesanchez1815** [nih-cfde/training-and-engagement#453] [RNA - Seq] [answered]
When looking at someone else’s data, say from GEO, do you have recommendations on how to gauge whether the experiment design is good or not? Are there rules or is it mostly qualitative?

**Comment by ACharbonneau** [comment, marked as answer]
I don't know of any set of good rules. In general, I'd look for blocking metadata at a datapoint level and detailed protocol information. If they don't give you information like lanes, or prep details for sequencing, or if they only report ranges for metadata it's going to be very hard to reuse.
